core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.html

```diff
@@ -1 +1,23 @@
-<p>dot-ema works!</p>
+<iframe
+    #iframe
+    [title]="host"
+    src="http://localhost:3000/test"
+    height="100%"
+    width="100%"
+    style="border: none"></iframe>
+<p-dialog
+    #dialog
+    [(visible)]="visible"
+    [style]="{ height: '90vh', width: '90vw' }"
```

Comment:
The primeng dialogs have a feature to make them full screen... can we try that?

Reply:
The property is to make them maximizable, you have to click again to make it fullscreen.